core/thegraph/state.go

```diff
@@ -365,3 +392,13 @@ func convertIndexedOperatorInfoGqlToIndexedOperatorInfo(operator *IndexedOperato
 		Socket:   string(operator.SocketUpdates[0].Socket),
 	}, nil
 }
+
+// Computes a cache key for the operator state cache. The cache key is a
+// combination of the block number and the quorum IDs. Note: the order of the
```

Comment:
I don't know if this is a problem but noticed that the cache key is dependent on the order you input the quorum ids.

Reply:
I think it's a unordered set, so you may eliminate the ordering effect by sorting the them

Reply:
Sounds good I'll add sorting.